common/autoware_lanelet2_utils/include/autoware_lanelet2_utils/intersection.hpp

```diff
@@ -0,0 +1,72 @@
+// Copyright 2025 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef AUTOWARE_LANELET2_UTILITY__INTERSECTION_HPP_
+#define AUTOWARE_LANELET2_UTILITY__INTERSECTION_HPP_
+
+#include <lanelet2_core/Forward.h>
+
+#include <optional>
+
+namespace autoware::lanelet2_utility
```

Comment:
Please change the namespace as well, just like the package name.
```suggestion
namespace autoware::lanelet2_utils
```

Reply:
Can you look at my comment ? I'd like to fix this in other PR for simplicity.